content/chat/typing.textile

```diff
@@ -5,3 +5,99 @@ product: chat
 languages:
   - javascript
 ---
+
+Signalling when a chat participant is typing has become a standard practice in various chat applications. This subtle yet valuable layer of interactivity within a chatroom can be seamlessly implemented. The typing indicator offers real-time functionality to promptly update the status of each chat participant's typing activities.
+
+<img src="/images/products/chat/typing.png" alt="Example of a chat message input box, above this has two names of other chat participants who are currently typing.">
```

Comment:
Image doesn't link up in Heroku

Reply:
https://github.com/ably/docs/pull/2090/commits/e38270283801b70f1719b0b10cd9ea6aab868339